pandas/io/tests/test_packers.py

```diff
@@ -496,6 +517,46 @@ def test_compression_blosc(self):
             assert_frame_equal(self.frame[k], i_rec[k])
 
 
+class TestMsgpack():
```

Comment:
what I mean is that doesn't actually test anything (as it doesn't inherit from TestCase) and would normally serve as a mix in for other test classes - but I don't see it being included anywhere - so what is its purpose?